jcstress-wrapper/src/main/kotlin/org/jetbrains/litmuskt/JCStressRunner.kt

```diff
@@ -58,18 +61,83 @@ class JCStressRunner(
             "target/jcstress.jar",
             *(jcsParams + jcstressFreeArgs),
             "-t",
-            test.javaClassName,
+            tests.joinToString("|") { "(${it.javaClassName})" },
         )
             .directory(jcstressDirectory.toFile())
             .redirectOutput(ProcessBuilder.Redirect.INHERIT)
             .redirectError(ProcessBuilder.Redirect.INHERIT)
             .start()
 
-        return {
+        return handle@{
             jcs.waitFor()
+            if (jcs.exitValue() != 0) error("jcstress exited with code ${jcs.exitValue()}")
+            return@handle tests.map { test -> parseJCStressResults(test) }
+        }
+    }
+
+    override fun <S : Any> startTest(test: LitmusTest<S>, params: LitmusRunParams): () -> LitmusResult {
+        val handle = startTests(listOf(test), params)
+        return { handle().first() }
+    }
+
+    private fun parseJCStressResults(test: LitmusTest<*>): LitmusResult {
```

Comment:
Please add a comment with an example of JCStress output you want to parse here. 

No need to paste the whole `html`, just a "schema" of the document you want to parse.

Reply:
:ok_hand: